controllers/games-controllers.js

```diff
@@ -29,4 +29,17 @@ exports.getReviewById = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+};
+
+exports.getCommentsByReviewId = (req, res, next) => {
+    const {review_id} = req.params;
+    const commentsPromise = fetchCommentsByReviewId(review_id)
+    const reviewPromise = fetchReviewById(review_id)
+    Promise.all([commentsPromise, reviewPromise])
```

Comment:
Very clever to differentiate between invalid article = [] and valid article with no comments = []. Well done! 🎖️